x-pack/plugins/index_management/public/application/sections/home/template_list/template_details/tabs/tab_summary.tsx

```diff
@@ -192,6 +193,21 @@ export const TabSummary: React.FunctionComponent<Props> = ({ templateDetails })
             <EuiDescriptionListDescription>
               {version || version === 0 ? version : i18nTexts.none}
             </EuiDescriptionListDescription>
+
+            {/* Allow auto create */}
+            {isLegacy === false && (
```

Comment:
I think we should also check if `isLegacy` is undefined, it is not always set to `false` for "non-legacy" templates, and the property is then not displayed in the flyout